src/main/java/org/example/homework4/ProductCostCalculator.java

```diff
@@ -0,0 +1,31 @@
+package org.example.homework4;
+
+public class ProductCostCalculator {
+    public static void main(String[] args) {
+        // price of the product
+        double unitPrice = 10.0;
+        // Quantity
+        int quantity = 55;
+
+        double totalCost = calculateTotalCost(unitPrice, quantity);
+        System.out.println("Total cost of the product: " + totalCost);
+    }
+
+    private static double calculateTotalCost(double unitPrice, int quantity) {
+        double totalCost;
+
+        if (quantity > 80)
+            totalCost = unitPrice * quantity * 0.87;
```

Comment:
a lot of duplications. this is correct, but consider inside if-else construction defining a `discount` value
and after it have one place for calculation of `totalCost`